tests/unit/amo/components/TestHomeHeroBanner.js

```diff
@@ -96,4 +102,37 @@ describe(__filename, () => {
       label: firstHeroTitle,
     });
   });
+
+  it('does not send a tracking event on first render when config experiment is disabled', () => {
```

Comment:
should we have a test case for the variant - to show it's null (or undefined as suggested above) somewhere here?

Reply:
if disabled..

Reply:
mmh, probably yeah

Reply:
added, thanks!